docs/user/power.md

```diff
@@ -0,0 +1,64 @@
+### Statistical Power Plots
+
+`plot_pwr0_statistic` simply plots statistical calculation of `pwr0`, lag 0, in the loaded in RAWACF file. 
+
+Basic code to plot lag 0, will plot the mean of `pwr0` for all frequencies at beam 0, from a RAWACF file would look like:
+```python
+import matplotlib.pyplot as plt
+import pydarn
+
+file = "20180101.0000.01.rkn.rawacf"
+sdarn_read = pydarn.SuperDARNRead(file)
+rawacf_data = sdarn_read.read_rawacf()
+
+pydarn.ACF.plot_acfs(rawacf_data)
+plt.show()
+```  
+
+![](../imgs/power_plot1.png)
+
+You also have access to numerous plotting options:
+
+
+| Parameter                  | Action                                                   |
+| -------------------------- | -------------------------------------------------------- |
+| beam_bum=0                 | beam number to plot                                      |
+| min_frequency              | minimum Frequency (inclusive) to plot                    |
+| max_frequency              | maximum Frequency (inclusive) to plot                    |
+| split_frequency            | specific frequency to look for or split between          |
+| statistical_method=np.mean | the statistical calculation to apply to the `pwr0` array |
+
+If you want display only one frequency set `min_frequency = max_frequency`:  
+
+```python
+import pydarn
+import matplotlib.pyplot as plt 
+
+rawacf_file = 'data/20201001.0200.00.sas.0.rawacf.hdf5'
+rawacf_data = pydarn.SuperDARNRead().read_borealis(rawacf_file)
```

Comment:
I don't think there was any documentation on read_borealis so might need an info box to say 'to read borealis data we use read_borealis, see more on this at pydarnio docs'?

Reply:
No this is something I implemented in pyDARN ... we probably should add it in your IO docs.